src/minimega/vm.go

```diff
@@ -568,6 +581,8 @@ func (vm *BaseVM) Info(field string) (string, error) {
 		return vm.Namespace, nil
 	case "state":
 		return vm.State.String(), nil
+	case "statetime":
```

Comment:
Used `time.Since` since @bjwrigh asked for something like `uptime`, which reports a duration?